src/utils/tests/wbsNumSort.test.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { wbsNumSort } from '../../components/projects/projects-table/projects-table/projects-table';
+
+describe('sort wbs numbers', () => {
+  it('equal wbsNums, asc', () => {
+    const a = '0.0.0';
+    const b = '0.0.0';
+
+    expect(wbsNumSort(a, b, 'asc')).toEqual(0);
+  });
+
+  it('equal wbsNums, desc', () => {
+    const a = '0.0.0';
+    const b = '0.0.0';
+
+    expect(wbsNumSort(a, b, 'desc')).toEqual(0);
+  });
+
+  it('equal wbsNums with different values for each part, asc', () => {
+    const a = '0.1.2';
+    const b = '0.1.2';
+
+    expect(wbsNumSort(a, b, 'asc')).toEqual(0);
+  });
+
+  it('equal wbsNums with different values for each part, desc', () => {
+    const a = '24.36.12';
+    const b = '24.36.12';
+
+    expect(wbsNumSort(a, b, 'desc')).toEqual(0);
+  });
+
+  it('a.car < b.car, asc', () => {
+    const a = '0.7.12';
+    const b = '1.56.44';
+
+    expect(wbsNumSort(a, b, 'asc')).toEqual(1);
+  });
+
+  it('a.car > b.car, asc', () => {
```

Comment:
The test body here matches the previous test but has a different name. Did you intend to change the test body to test something similar but different?

Reply:
Yes, some of the tests had an incorrect expected value with the wrong sign. This also revealed a bug within the sorting function which for some reason didn't make itself apparent at all when the app was running.